tests/containers/bci_tests.pm

```diff
@@ -107,13 +104,21 @@ sub run {
     assert_script_run('tox -e build', timeout => 600);
 
     # Run the tests for each environment
-    my $errors = 0;
+    my $error_count = 0;
     for my $env (split(/,/, $test_envs)) {
         record_info("Test: $env");
-        script_run("tox -e $env", timeout => $bci_timeout);
+        my $ret = script_run("tox -e $env", timeout => $bci_timeout);
+        if ($ret != 0) {
+            $error_count += 1;
+            record_info('Meeeh', "There was a problem running the test: $env");
+        }
     }
 
     $self->parse_logs();
+
+    # Record soft failure not to block container image release process record
+    # soft failure. When BCI tests repo is stable enough it can be changed to "die"
+    record_soft_failure("$error_count tests failed.") if ($error_count gt 0);
```

Comment:
this is smelly IMO.
you shouldnt need to do anything like that in first place. 
`tox` should produces those results and from this point of you the test looks wrong.

Reply:
now we have a `die`, as it should be.

Reply:
tox returns 0 or 1 and I dont want to use assert_script_run for those cause I want to run all of them and see what happens. If I use assert_script_run for tox commands, if 1 fails it will stop the test.

Reply:
for ref: https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/419. thanks @jlausuch for that